modules/authorization/proc_enabling_transitive_parent_groups.adoc

```diff
@@ -0,0 +1,22 @@
+= Enabling transitive parent groups
+
+By default, {product} does not resolve indirect parent groups during authentication.
+In this case, with the following group hierarchy, the `user_alice` user is only a member of the `group_developers` group:
+
+----
+group_admin
+  └── group_developers
+    └── user_alice
+----
+
+To accomodate complex group hirearchy, you can configure {product-short} to add indirect parent groups to the user ownership entities.
```

Comment:
I think we should mention that this config only impacts how the `$ownerRefs` alias is handled

``` suggestion
To support multi-level group hierarchies when using the $ownerRefs alias, you can configure {product-short} to include indirect parent groups in the user’s ownership entities.
```